examples/gitlab-centos7-atomicapp/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM projectatomic/atomicapp:0.1.3
+
+MAINTAINER Navid Shaikh <nshaikh@redhat.com>
+
+LABEL io.projectatomic.nulecule.specversion="0.0.2" \
+      io.projectatomic.nulecule.providers="kubernetes"\
+      Build="docker build --rm --tag swordphilic/gitlab-atomicapp ."
+
+ADD /Nulecule /Dockerfile README.md gpl-3.0.txt answers.conf.sample /application-entity/
```

Comment:
If you add `params` section with all available parameters they will appear in `answers.conf.sample` with default values after `atomic install`:-)